templates/CRM/Core/Calendar/Rss.tpl

```diff
@@ -27,8 +27,8 @@
 {/if}
 {if $event.start_date}{ts}When{/ts}: {$event.start_date|crmDate}{if $event.end_date} {ts}through{/ts} {strip}
         {* Only show end time if end date = start date *}
-        {if $event.end_date|date_format:"%Y%m%d" == $event.start_date|date_format:"%Y%m%d"}
-            {$event.end_date|date_format:"%I:%M %p"}
+        {if $event.end_date|crmDate:"%Y%m%d" == $event.start_date|crmDate:"%Y%m%d"}
+            {$event.end_date|crmDate:"%I:%M %p"}
```

Comment:
This is also slightly different, e.g. `12:13 PM` vs `12:13 pm`. But I don't think that would be confusing, and it's unlikely people have automated scripts that are parsing this or something.